x-pack/legacy/plugins/siem/index.ts

```diff
@@ -153,6 +153,14 @@ export const siem = (kibana: any) => {
         },
       },
       mappings: savedObjectMappings,
+      savedObjectSchemas: {
+        'case-workflow': {
+          indexPattern: '.case-testing-ground', // TODO: Change this name and use kibana.yml settings to override it.
```

Comment:
It looks like that you should have this attribute `isNamespaceAgnostic` too. It seems to be a required field from [here](https://github.com/elastic/kibana/blob/master/src/core/server/saved_objects/schema/schema.ts#L23)